content/docs/inventory/stock_takes.md

```diff
@@ -57,62 +55,132 @@ Let's start a new stocktake. To do so, tap on the `New Stocktake` button in the
+
+![Stocktake edit window](/docs/inventory/images/stocktake_edit.png)
+
+You can use the `Add batch (+)` button to add more batches of a particular item during your stocktake. This will add a new blank row, where you can enter batch information and the counted number of packs.
+
+![Stocktake add batch](/docs/inventory/images/stocktake_add_batch.png)
+
+<div class="note">You are not able to edit the pack size of stocktake rows which are linked to an existing stock line. If you wish to repack, follow the instructions in <a href="/docs/inventory/stock-view/#repacking-stock">Repack</a>.</div>
+
 ### Entering reasons
 
 If you have [inventory adjustment options](https://docs.msupply.org.nz/preferences:options?s[]=reasons) configured in your central server, then you are required to enter a reason when the `counted packs` specified does not match the snapshot packs.
 
-For example, entering `11` for the counted quantity of Abacavir Oral Solution, will add a red \* to the right of the `Reason` input and show the negative inventory adjustment reasons:
+For example, after entering `95` for the counted quantity of this batch Amoxicillin 250mg tabs, a red \* will appear to the right of the `Reason` input, and you will need to select one of the negative inventory adjustment reasons:
 
 ![Stocktake reasons](/docs/inventory/images/stocktake_reasons.png)
 
 If you try to save the stocktake row without entering a reason you will see an error:
 
-![Stocktake reasons](/docs/inventory/images/stocktake_reasons_error.png)
+![Stocktake reasons error](/docs/inventory/images/stocktake_reasons_error.png)
 
 and the stocktake row that requires a reason to be entered will be highlighted in red as shown below.
 
-![Stocktake reasons](/docs/inventory/images/stocktake_reasons_error_highlight.png)
+![Stocktake reasons error highlight](/docs/inventory/images/stocktake_reasons_error_highlight.png)
 
-### Printing Stocktake sheet
+### Adding items
+
+If an item was not included in the generated stocktake rows when you created your stocktake, you can add it manually. Click the `Add Item` button in the top right of your screen.
+
+![Stocktake add item](/docs/inventory/images/stocktake_add_item.png)
+
+An `Add Item` window will appear, where you can select the item you would like to add to your stocktake.
+
+![Stocktake add item window](/docs/inventory/images/stocktake_add_item_modal.png)
+
+Once you select an item, stocktake rows will be generated for any batches of that item with stock, and you can carry out your stocktake data entry as above.
+
+![Stocktake add item with batches](/docs/inventory/images/stocktake_newitem_with_batches.png)
+
+If there are no batches with stock for that item, your batch list will be blank. You can use the `Add batch (+)` button to add a new blank row, enter the batch information and the counted number of packs.
```

Comment:
```suggestion
If there are no batches with stock for that item, your batch list will be blank. You can use the `Add batch (+)` button to add a new blank row where you can enter the batch information and the counted number of packs.
```

Sounded weird to me for some reason. Tried to make it sound smoother but maybe it isn't? 👀